assets/scss/style_variations.scss

```diff
@@ -83,3 +83,273 @@ aside#secondary > section{
 .nhsuk-icon__arrow-right-circle{
   vertical-align: middle;
 }
+
+
+/* Events Calendar Styles */
```

Comment:
can all of this be moved to its own scss file for standardising - the learndash and gravity plugin styling is in their own files. This just makes it easier for future developers to find where styles are being generated from.